cfme/services/myservice/ssui.py

```diff
@@ -206,10 +190,10 @@ def update(self, updates):
     view.fill_with(updates, on_change=view.save_button, no_change=view.cancel_button)
     view.flash.assert_no_error()
     view = self.create_view(DetailsMyServiceView, override=updates)
-    # TODO - remove sleep when BZ 1518954 is fixed
-    time.sleep(10)
-    assert view.notification.assert_message(
-        "{} was edited.".format(self.name))
+    message_present, _ = wait_for(view.notification.assert_message,
+                                  func_args=[f'{self.name} was edited.'],
+                                  timeout=10)
+    assert message_present
```

Comment:
this is redundant since you alrady check this in wait_for and it will fail with exception otherwise.